sklearn/tests/test_pipeline.py

```diff
@@ -330,6 +332,37 @@ def test_pipeline_methods_pca_svm():
     pipe.score(X, y)
 
 
+@pytest.mark.filterwarnings('ignore: default contamination parameter')  # 0.22
```

Comment:
You don't need this line anymore. We merge the PR which remove the deprecation for 0.22

Reply:
Ok.